zulipterminal/ui_tools/views.py

```diff
@@ -1077,12 +1077,15 @@ def __init__(self, controller: Any, stream_id: int) -> None:
         self.stream_id = stream_id
         self.controller = controller
         stream = controller.model.stream_dict[stream_id]
+        weekly_msg_count = controller.model.stream_dict[stream_id][
+            'stream_weekly_traffic']
         keys = ', '.join(map(repr, keys_for_command('STREAM_MEMBERS')))
         title = '# {}'.format(stream['name'])
         desc = stream['description']
         stream_info_content = [('', [desc]),
-                               ('Stream Info', [
-                                   ('Estimated messages per week', str(93)),
+                               ('Stream Details', [
+                                   ('Weekly Message Count',
+                                    str(weekly_msg_count)),
```

Comment:
All of this is different from the previous commit? Let's just add all the message count details into this commit.